src/main/java/com/jvm_bloggers/frontend/public_area/jvm_poland_slack/JvmPolandSlackPage.html

```diff
@@ -0,0 +1,23 @@
+<html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.5-strict.dtd">
```

Comment:
Could you provide a screenshot how it looks like?

Reply:
@tdziurko below
I didn't know how to fill that empty space :stuck_out_tongue: 
![image](https://user-images.githubusercontent.com/17989854/75369116-19075780-58c3-11ea-80a2-4bec3e12361f.png)